runtime/compiler/env/VMJ9.cpp

```diff
@@ -3078,6 +3078,17 @@ static TR::ILOpCodes udataCmpEqOpCode(TR::Compilation * comp)
       }
    }
 
+TR::Node *
+TR_J9VMBase::testIsClassValueType(TR::Node *j9ClassRefNode)
```

Comment:
Just curious, would it be worth of making this API more generic to test other class flags instead of only `J9ClassIsValueType`? 

`TR_J9VMBase::testClassFlag(TR::Node *j9ClassRefNode, int32_t flag)
`

Reply:
I thought about that, but I figured part of the point was to hide the reference to `J9ClassIsValueType`, so I didn't include a flags argument.  I could certainly introduce a `testClassFlag` method and have `testIsClassValueType` call it from within `TR_J9VMBase`, so that at least the code that sets up access to the class's flags field would be common for any other methods that would need to dynamically test the flags.

Reply:
Annabelle @a7ehuo, I just added commit c9c45ff60 that addresses this comment.  If you and Vijay @vijaysun-omr are OK with it, I can squash the two commits.

Reply:
LGTM

Reply:
Thank you both for reviewing.  I've squashed the two commits down to https://github.com/eclipse/openj9/pull/12126/commits/c9d6801068f9fd6a7df84c829d0bb4a2b5c692da